notebooks/curation_api/python/src/dataset.py

```diff
@@ -31,7 +30,7 @@ def get_identifier_type_and_value(identifier: str) -> Tuple[str, str]:
         identifier_type = "id"
     else:
         # CURATOR_TAG_PREFIX_REGEX is superfluous; leaving in to match lambda handler code; may use later
-        matched = re.match(f"({UUID_REGEX}|{CURATOR_TAG_PREFIX_REGEX})\\.{EXTENSION_REGEX}$", identifier)
+        matched = re.match(f"({UUID_REGEX}|{CURATOR_TAG_PREFIX_REGEX})$", identifier)
```

Comment:
nit: without the EXTENSION_REGEX check to differentiate it from the earlier check, we don't need to check for the UUID_REGEX again here right?

Reply:
^ if that is true, we can probably simplify or remove most of the logic in this function, like the groupdict check for id below